cluster/addons/dns/skydns-rc.yaml.base

```diff
@@ -104,20 +111,20 @@ spec:
         resources:
           # keep request = limit to keep this container in guaranteed class
           limits:
-            cpu: 10m
+            cpu: 20m
             memory: 50Mi
           requests:
-            cpu: 10m
+            cpu: 20m
             # Note that this container shouldn't really need 50Mi of memory. The
             # limits are set higher than expected pending investigation on #29688.
             # The extra memory was stolen from the kubedns container to keep the
             # net memory requested by the pod constant.
             memory: 50Mi
-        args:
-        - -cmd=nslookup kubernetes.default.svc.__PILLAR__DNS__DOMAIN__ 127.0.0.1 >/dev/null && nslookup kubernetes.default.svc.__PILLAR__DNS__DOMAIN__ 127.0.0.1:10053 >/dev/null
-        - -port=8080
-        - -quiet
+        command: ["/bin/sh","-c"]
+        args: ["/exechealthz -cmd='nslookup kubernetes.default.svc.__PILLAR__DNS__DOMAIN__ 127.0.0.1 >/dev/null' -port=8082 -quiet & /exechealthz -cmd='nslookup kubernetes.default.svc.__PILLAR__DNS__DOMAIN__ 127.0.0.1:10053 >/dev/null' -port=8080 -quiet"]
```

Comment:
We should not do this - if one of them dies it will not get restarted

We should either run two whole containers or make one container able to handle both (as you suggested)